ci_framework/roles/os_net_setup/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+- name: get openstack admin credentials from k8s
+  block:
+  - name: Get clouds.yaml configuration
+    kubernetes.core.k8s_info:
+      kubeconfig: "{{ k8s_kubeconfig }}"
+      context: "{{ k8s_context }}"
+      kind: ConfigMap
+      namespace: openstack
+      name: openstack-config
+    register: clouds_yaml
+
+  - name: Get the OSP admin password
+    kubernetes.core.k8s_info:
+      kubeconfig: "{{ k8s_kubeconfig }}"
+      context: "{{ k8s_context }}"
+      kind: Secret
+      namespace: openstack
+      name: openstack-config-secret
+    register: osp_secret
+  vars:
```

Comment:
it would be more readable if that `vars` block was moved between the `name` and `block` entry.

Reply:
moved